plugins/org.opencast.usertracking.MatomoSaverPlugIn/usertracking_matomo_saver.js

```diff
@@ -69,6 +69,12 @@ paella.addPlugin(function() {
       }
     }
 
+    loadTitle() {
+      var title = paella.player.videoLoader.getMetadata() && paella.player.videoLoader.getMetadata().title;
+      document.title = title || document.title;
```

Comment:
You don't have to change the `document.title`.

Please return the `title' variable.
```javascript
return title;
```